src/dbnode/persist/fs/seek_manager.go

```diff
@@ -111,19 +118,27 @@ type seekersAndBloom struct {
 	volume      int
 }
 
-// borrowableSeeker is just a seeker with an additional field for keeping track of whether or not it has been borrowed.
+// borrowableSeeker is just a seeker with an additional field for keeping
+// track of whether or not it has been borrowed. A seeker is essentially
+// a wrapper around file descriptors around a set of files, allowing for
+// interaction with them. We can ask a seeker for a specific time series, which
+// will then be streamed out from the according data file.
 type borrowableSeeker struct {
 	seeker     ConcurrentDataFileSetSeeker
 	isBorrowed bool
 }
 
+// seekersByTime contains all seekers for a specific shard, accessible by
+// blockStart. The accessed field allows for pre-caching those seekers.
 type seekersByTime struct {
 	sync.RWMutex
 	shard    uint32
 	accessed bool
 	seekers  map[xtime.UnixNano]rotatableSeekers
 }
 
+// rotatableSeekers is a wrapper around seekersAndBloom that allows for rotating
+// out stale seekers.
```

Comment:
"out stale seekers. This is required so that the active seekers can be rotated to inactive while the seeker manager waits for any outstanding stale seekers to be returned."

Reply:
Added your suggestion.